Samples/I2C/I2C_LSM6DS3_HighLevelApp/README.md

```diff
@@ -22,7 +22,7 @@ The sample uses the following Azure Sphere libraries:
 
 - [ST LSM6DS3](https://www.st.com/en/mems-and-sensors/lsm6ds3.html)
 
-   **Note:** By default, this sample is configured to use an external LSM6DS3, and isn't configured to use the onboard sensors found on some development boards, such as the LSM6DS0 on the Avnet Starter Kit. This sample uses ISU1 on the MT3620 board; however, you can use another ISU by adjusting the wiring, the code, and the application manifest.
+   **Note:** By default, this sample is configured to use an external LSM6DS3 device.  The Avnet Starter Kits include an on-board [LSM6DS0](https://www.st.com/en/mems-and-sensors/lsm6dso.html) sensor.  This example will work on the Avnet Starter Kit (to read the on-board LSM6DS0 device) with some minor modifications, see the detailed instructions in the section titled "Changes required to use the Avnet MT3620 Starter Kit and its built-in LSM6SDO accelerometer."  Since the LSM6DS3 and LSD6DS0 devices would be on the same I2C interface (ISU2), and the two devices contain overlapping register address', the Avnet Starter Kit is not able to use the external LSM6DS3 device.
```

Comment:
If SAMPLE_LSM6DS3_I2C is mapped to ISU1, we would avoid this issue.

Reply:
@anpintil, the issue with mapping SAMPLE_LSM6DS3_I2C to ISU1 is that the ISU1 SCL and SDA signals are not pinned out on the Avnet kits.

What about the idea of removing all references to SAMPLE_LSM6DS3_I2C from the Avnet Hardware Definition files.  If we made this change, then the I2C example would not build when the Avnet kits were selected.  Then at least the user would see that there is an issue.  Hopefully they would look at the documentation at that point and see what changes need to be made.

With the project configured the way it is now, the user will expect the LSM6DS3 device to work, but it never will because of the LSMDS0 conflict.  When they debug the issue they will discover that the Who Am I response from the device is from the LSM6DS0 device, not the LSM6DS3 as expected.